src/httpauth/digest.c

```diff
@@ -16,21 +16,37 @@
 #include <re_httpauth.h>
 
 
+enum {
+	CNONCE_NC_SIZE = 9, /* 8 characters + '\0' */
+};
```

Comment:
is this used ?

Reply:
you are right, its not used anymore. it was to store the cnonce and nc as strings
it is removed now